pandas/_libs/window/meson.build

```diff
@@ -3,7 +3,6 @@ py.extension_module(
     ['aggregations.pyx'],
     cython_args: ['-X always_allow_keywords=true'],
     include_directories: [inc_np, inc_pd],
-    dependencies: [py_dep],
```

Comment:
Can you do the same for ``pandas/_libs/meson.build`` and ``pandas/_libs/tslibs/meson.build``?

Reply:
Unless I misunderstanding the request, these other `meson.build` files do not contain the problematic construct.